conf/broker.conf

```diff
@@ -1414,4 +1414,7 @@ tlsEnabled=false
 
 # Enable Key_Shared subscription (default is enabled)
 # @deprecated since 2.8.0 subscriptionTypesEnabled is preferred over subscriptionKeySharedEnable.
-subscriptionKeySharedEnable=true
\ No newline at end of file
+subscriptionKeySharedEnable=true
+
+#enable or disable strict bookie affinity
+strictBookieAffinityEnabled=false
```

Comment:
This is `### --- Deprecated settings --- ###`
Please move to a proper position.

Reply:
Thx, PTAL again. @Jason918